Dataset.py

```diff
@@ -596,6 +633,8 @@ def initialize(self):
 
   def get_times(self, sorted_seq_idx):
     """
+    TODO: is used only in eval mode, and implemented only by the CachedDataset
```

Comment:
Yes, maybe. This could be for historical reasons. I don't remember anymore.
But what is your TODO about? What do you want to do about it?